src/networkcloud/azext_networkcloud/aaz/operations/__init__.py

```diff
@@ -0,0 +1,4 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Please move the `operations` folder out of `aaz`.

Reply:
Updated.